tests/functional/modules/test_zos_job_submit_func.py

```diff
@@ -158,6 +158,18 @@
 /*
 //SYSUT2   DD SYSOUT=*
 //
+""",
+    "Autoescape": """//{{ pgm_name }}    JOB (T043JM,JM00,1,0,0,0),{{ parameter }},CLASS=R,
+//             MSGCLASS=X,MSGLEVEL=1,NOTIFY=S0JM
```

Comment:
In the issue related users complained both about `&` and `'` could you add those as part of the tests?

Reply:
oh I see single quotes is added down there so just `&`